src/smc-util-node/test/misc_node-test.coffee

```diff
@@ -48,4 +48,25 @@ describe "sanitizing HTML", ->
+
+describe 'check sales tax work', ->
+    sales_tax = misc_node.sales_tax
+
+    it 'knows about 98122', ->
+        expect(sales_tax('98122')).toBe 0.10100
```

Comment:
That seems inconsistent with the rest of our tests.
They are largely of the form
`thing.should.be "value"`

eg.
`l({}).should.be.exactly 0`
`should.be.exactly 47`
`expect(ret).toBe "<p>hello</p>"`